flexinput/src/main/java/com/lytefast/flexinput/model/Photo.kt

```diff
@@ -22,6 +25,10 @@ class Photo : Attachment<String> {
 
   constructor(parcelIn: Parcel) : super(parcelIn)
 
+  @TargetApi(Build.VERSION_CODES.Q)
+  fun getThumbnailQ(contentResolver: ContentResolver, width: Int, height: Int) =
+      contentResolver.loadThumbnail(uri, Size(width, height), null)
```

Comment:
This is the new magic

Reply:
we need to manage cancellation if you cancel the job:
https://developer.android.com/reference/android/content/ContentResolver.html#loadThumbnail(android.net.Uri,%20android.util.Size,%20android.os.